pyscript.core/src/stdlib/pyweb/pydom.py

```diff
@@ -419,8 +446,13 @@ def create(self, type_, classes=None, html=None):
 
     def __getitem__(self, key):
         if isinstance(key, int):
-            indices = range(*key.indices(len(self.list)))
-            return [self.list[i] for i in indices]
+            out = []
+            for el in self._js.children:
+                out.append(Element(el))
+            return out[key]
+            # return [Element(el) for el in self._js.children][key]
+            # indices = range(*key.indices(len(self.list)))
+            # return [self.list[i] for i in indices]
 
         elements = self._js.querySelectorAll(key)
         if not elements:
```

Comment:
I know in Python empty lists would make this `true` but I have no idea if proxied NodeList static collections would do the same here ... worth checking, as this code might never be reached.

Reply:
This is ugly code that I changed to debug/make things work, definiitely improving this soon

Reply:
is today soon enough? :-)